packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -305,6 +305,11 @@ export async function unlinkLogin(account: AddressInfo) {
     auth: true,
     jwt: app.user.jwt,
   });
+  // remove deleted role from app.roles
+  app.roles.deleteRole({
+    address: account,
+    chain: account.chain.id,
+  })
```

Comment:
when a user leaves a community, this code will update `app.roles` and remove the deleted role (This was a different bug I would say, not related to the cleanup work)